src/optionsHandlers/handlers.ts

```diff
@@ -27,3 +32,12 @@ export const handleDryRunFlag = (): { dryRun: boolean } => {
 };
 
 export const handleDebugFlag = (): boolean => getInput(Flags.debug) === 'true';
+
+export const handleScriptPathFlag = async (): Promise<string> => {
+  const scriptPathInput: string = getInput(Flags.scriptPath);
```

Comment:
Should we resolve the path relative to `GITHUB_WORKSPACE`?

Reply:
Actions and shell commands execute in `/github/workspace` (GITHUB_WORKSPACE) directory, so `fs.exists` is able to check if the required file is present in the workspace.